public/sass/components/_gf-form.scss

```diff
@@ -139,8 +147,15 @@ $gf-form-label-margin: 0.25rem;
   }
 }
 
-.gf-form-select-wrapper + .gf-form-select-wrapper { 
-  margin-left: $gf-form-label-margin; 
+.gf-form--textarea {
+  align-items: flex-start;
+}
+
+input[type="number"].gf-natural-language-form {
```

Comment:
can you show me? I mean "gf-form-inline" is meant to be used when inlining multiple gf-form elements